### PR TITLE
Fix import of alternative vehicles

### PR DESCRIPTION
**Pull request by Gymnasiast**
Picked the best fit in case no direct equivalent was available.

Vertigo Views before:
scr159

Vertigo Views after:
scr160

Katie's Dreamland before:
scr158

Katie's Dreamland after: (Swans are also fixed)
scr161

### CODE REVIEW

**Comment by IntelOrca** [comment]
Using the rocket cars on junior track, looks so simple now... well done.

File: src/rct1.c [hunk 1]
```diff
@@ -2048,7 +2078,92 @@ static void rct1_import_ride(rct1_s4 *s4, rct_ride *dst, rct1_ride *src)
...
+			break;
+		case 82:
+			dst->subtype = 108;
+			break;
+		case 85:
+			dst->subtype = 109;
+			break;
+		default:
+			dst->subtype = src->type;
+	}
```
Comment (IntelOrca) [inline src/rct1.c]: Could you use an array / table instead of a switch?
Like RCT1DefaultObjectsRides and others are done.

File: src/rct1.c [hunk 2]
```diff
@@ -1072,7 +1076,126 @@ static const rct_object_entry RCT1DefaultObjectsRides[] = {
...
+	{ 0x00008000, { "BMFL    " }, 0 },
+	{ 0x00008000, { "CLIFT2  " }, 0 },
+	{ 0x00008000, { "BMRB    " }, 0 },
+	{ 0x00008000, { "UTCARR  " }, 0 },
+	{ 0x00008000, { "ARRT2   " }, 0 },
+	{ 0x00008000, { "SLCFO   " }, 0 },
+	{ 0x00008000, { "AML1    " }, 0 }
+};
+
+// A value of 255 means the default vehicle for the respective track type should be used.
```
Comment (IntelOrca) [inline src/rct1.c]: Use a constant for this, like
```
#define DEFAULT_VEHICLE_MAP 255
```

**Comment by Gymnasiast** [comment]
I changed it to use a constant, although I opted for a (IMO) more descriptive name.

**Pull request merged**